### DISCUSSION TITLE
Windowing CT

### DISCUSSION THREAD

**Discussion by hreso110100** [Project-MONAI/MONAI#6460] [General]
Hi is there some transform in MONAI which helps me to achieve CT windowing like in this post ? https://radiopaedia.org/articles/windowing-ct

**Comment by KumoLiu** [comment]
Hi @hreso110100, you could refer to this [discussion](https://github.com/Project-MONAI/tutorials/discussions/1036).
Hope it can help you, thanks!

**Reply by hreso110100** [reply]
thanks I will check it
